packages/core/src/components/Banner/Banner.tsx

```diff
@@ -103,42 +104,43 @@ export const HvBanner = ({
   );
 
   return (
-    <StyledSnackbar
-      style={anchorOriginOffset[`anchorOrigin${capitalize(anchorOrigin)}`]}
-      className={className}
-      id={id}
-      classes={{
-        root: open
-          ? clsx(bannerClasses.root, classes?.root)
-          : clsx(bannerClasses.rootClosed, classes?.rootClosed),
-        anchorOriginTopCenter: clsx(
-          bannerClasses.anchorOriginTopCenter,
-          classes?.anchorOriginTopCenter
-        ),
-        anchorOriginBottomCenter: clsx(
-          bannerClasses.anchorOriginBottomCenter,
-          classes?.anchorOriginBottomCenter
-        ),
-      }}
-      anchorOrigin={anchorOriginBanner}
-      TransitionComponent={SlideTransition}
-      open={open}
-      $isOpen={open}
-      transitionDuration={transitionDuration}
-      {...others}
-    >
-      <HvBannerContent
-        id={setId(id, "content")}
-        content={label}
-        variant={variant}
-        customIcon={customIcon}
-        showIcon={showIcon}
-        actions={actions}
-        actionsCallback={actionsCallback}
-        actionsPosition={actionsPosition}
-        onClose={onClose}
-        {...bannerContentProps}
-      />
-    </StyledSnackbar>
+    <ClassNames>
+      {({ css, cx }) => (
+        <Snackbar
+          id={id}
+          open={open}
+          className={cx(css(bannerClasses.root), className)}
```

Comment:
Some classes are missing. I also think that we should continue to use the `rootClosed` classe to maintain backward compatibility with the previous API and by doing so we don't need to have the `[`&.open`]` selector.

Maybe something like:

```
className={className}
classes={{
            root: open ? cx(bannerClasses.root, css(styles.root), classes?.root) : cx(bannerClasses.rootClosed, css(styles.rootClosed), classes?.rootClosed),
            anchorOriginTopCenter: cx(
              bannerClasses.anchorOriginTopCenter,
              css(styles.anchorOriginTopCenter),
              classes?.anchorOriginTopCenter
            ),
            anchorOriginBottomCenter: cx(
              bannerClasses.anchorOriginBottomCenter,
              css(styles.anchorOriginBottomCenter),
              classes?.anchorOriginBottomCenter
            ),
}}
```

Reply:
yeah, I wondered about that and thought I could improve, but I'll keep how it was like you suggested.

Reply:
Okay, great 🙏 

I believe you forgot to update the `className`. It's currently `className={cx(css(bannerClasses.root), className)}` and should be `className={className}`.